jaeger-core/build.gradle

```diff
@@ -19,6 +20,18 @@ dependencies {
     signature 'org.codehaus.mojo.signature:java16:1.1@signature'
 }
 
+apply plugin: 'com.github.johnrengelman.shadow'
```

Comment:
@pavolloffay I think we should because http libraries are extremely common and prone to conflict. 
Even in this project, without shading, the build breaks because `jaeger-zipkin` depends on `jaeger-core`, and the okhttp versions conflict (3.4.x vs 3.8.x). 
Also, the shaded artifact is published under a different classifier, so it wouldn't affect people who don't need it.